SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -341,7 +341,7 @@ Buffer {
 						startFrame = 0, leaveOpen = false, completionMessage;
 		// doesn't change my path
 		^["/b_write", bufnum, path,
-				headerFormat, sampleFormat, numFrames, startFrame,
+				headerFormat, sampleFormat, numFrames.asInt, startFrame.asInt,
```

Comment:
But, in general I'd prefer first to have a discussion about the int vs float scsynth API. I don't think that we should have to add `asInteger` all over the place.